launcher/game/pefile.py

```diff
@@ -824,7 +824,7 @@ def dump(self, indentation=0):
                         except exceptions.ValueError, e:
                             val_str += ' [INVALID TIME]'
                 else:
-                    val_str = ''.join(filter(lambda c:c != '\0', str(val)))
+                    val_str = ''.join([c for c in str(val) if c != '\0'])
```

Comment:
This seems to just be converting to a string and then stripping NULs.
It would probably be more straightforward and easier to understand as:
```python
                    val_str = str(val).replace('\0', '')
```
It might also be faster as it does not have to convert to lists of individual character substrings.